midend/parserUnroll.cpp

```diff
@@ -674,6 +694,22 @@ class ParserSymbolicInterpreter {
         parser = structure->parser;
         hasOutOfboundState = false;
     }
+
+    /// generate call OutOfBound
```

Comment:
Creates a new state that immediately transitions to the "outOfBoundsState" state.